### ISSUE TITLE
[Bug]: Check constraints are not updated when columns are renamed.

### ISSUE THREAD

**Issue by pchemguy** [sqlitebrowser/sqlitebrowser#3275]
### What did you do?

1. Created a new in-memory database
2. Executed SQL

~~~sql
CREATE TABLE "paths" (
	"prefix"	TEXT,
	"name"	TEXT NOT NULL,
	"paths"	TEXT GENERATED ALWAYS AS ("prefix" || "name") VIRTUAL,
	CONSTRAINT "chk_prefix" CHECK(length("prefix" || "name") > 0)
);
~~~

3.  Renamed columns "prefix"/"name".

### What did you expect to see?

Properly updated CHECK constraint.

### What did you see instead?

CHECK constraint is not updated.

### DB4S Version

3.12.99 (nightly)

### What OS are you seeing the problem on?

Windows

### OS version

Windows 10

### Relevant log output

```shell
~~~sql
CREATE TABLE "sqlb_temp_table_15" (
	"prefix1"	TEXT,
	"name1"	TEXT NOT NULL,
	"paths"	TEXT GENERATED ALWAYS AS ("prefix" || "name") VIRTUAL,
	CONSTRAINT "chk_prefix" CHECK(length("prefix" || "name") > 0)
);
~~~
```

### Prevention against duplicate issues

- [X] I have searched for similar issues

**Comment by MKleusberg** [comment]
Yes, you're right. This should be fixed. It's probably not happening anytime soon though because it would require us to fully parse and store the check and generated as expressions which we're currently not doing.
